beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandler.java

```diff
@@ -159,6 +160,11 @@ public ValidatorApiHandler(
     this.proposersDataManager = proposersDataManager;
   }
 
+  @Override
+  public SafeFuture<Optional<SyncingStatus>> getSyncingStatus() {
+    return SafeFuture.failedFuture(new UnsupportedOperationException("Not yet implemented"));
```

Comment:
Typically we want the implementation here to be the canonical one and then the REST API basically just delegates to this. That way the remote and internal VCs are getting the same behaviour because it winds up at the same code.

So in this case we'd ideally want to take the business logic that's currently used for the `GetSyncing` API and put it here then `GetSyncing` would delegate to this. I'm not sure what that will wind up looking like so maybe it will work out better in this case to have this delegate to the same thing `GetSyncing` uses but I suspect that means `ValidatorApiHandler` which is a fairly internal kind of class winds up needing to have access to more REST API specific classes which isn't great.

You'll need to play around and see what you think works best. :)

Reply:
Actually the other option is that we don't add this method `ValidatorApiHandler` at all and just make it available only on `RemoteValidatorApiHandler` or something like that. If it's only useful for remote validators it would be nice to avoid needing to add it here. Again, depends on the trade offs involved.

Reply:
Definitely agree, that will be cleaner and makes more sense, since it will only be used there.